test/aries-js-worker/test/vdri/vdri.js

```diff
@@ -54,62 +37,66 @@ describe("VDRI", function () {
         )
     })
 
-    it("create public DID using VDRI in WASM and REST Client mode", async function () {
-        if (!agents || agents.length === 0) {
-            throw new Error("invalid agents initialized")
-        }
-
-        for (let i = 0; i < agents.length; i++) {
-            await agents[i].vdri.createPublicDID({
-                method: "sidetree",
-                header: '{"alg":"","kid":"","operation":"create"}'
-            })
-        }
-    })
-
     it("Alice stores the did generated by her", async function () {
-        for (let i = 0; i < agents.length; i++) {
+        for (let i = 0; i < 1; i++) {
+            let resp;
+            for (let x = 0; x < 10; x++) {
+                try {
+                    let id=didID
+                    if (i===1){
+                        id=window.btoa(didID)
+                    }
+
+                    resp = await agents[i].vdri.resolveDID({id: didID})
+                } catch (e) {
+                    assert.fail(e.message);
+                    if (!e.message.includes("DID does not exist")) {
+                        assert.fail(e.message);
+                    }
+                    await new Promise(r => setTimeout(r, 1000));
+                }
+            }
             await agents[i].vdri.saveDID({
                 name: didName,
-                did: doc
+                did: resp.did
             })
         }
     })
 
-    it("Alice retrieves the did from store",  async function () {
-        let errors = "Alice didn't retrieve did: ";
-
-        for (let i = 0; i < agents.length; i++) {
-            try {
-                await agents[i].vdri.getDID({id: didID})
-            }catch (e) {
-                errors += e.message;
-                continue
-            }
-
-            return
-        }
-
-        throw new Error(errors)
-    })
-
-    it("Alice validates that she has the did", async function () {
-        for (let i = 0; i < agents.length; i++) {
-            let resp;
-            try {
-                resp = await agents[i].vdri.getDIDRecords()
-            }catch (e) {
-                continue
-            }
-
-            for (let j = 0 ;j<resp.result.length;j++){
-                if (didID === resp.result[j].id){
-                    assert.equal(didName, resp.result[j].name)
-                    return
-                }
-            }
-        }
-
-        throw new Error("Alice doesn't have did")
-    })
+    // it("Alice retrieves the did from store",  async function () {
```

Comment:
why commented out

Reply:
I commented out by mistake. is fixed now